uuid/uuid_test.go

```diff
@@ -2,15 +2,23 @@ package uuid_test
 
 import (
 	"fmt"
+	"log"
+	"os"
 	"testing"
 	"time"
 
+	"github.com/google/uuid"
 	. "github.com/tarantool/go-tarantool"
-	_ "github.com/tarantool/go-tarantool/uuid" 
+	"github.com/tarantool/go-tarantool/test_helpers"
+	_ "github.com/tarantool/go-tarantool/uuid"
 	"gopkg.in/vmihailenco/msgpack.v2"
-	"github.com/google/uuid"
```

Comment:
This change fixes an ordering of imports, and it is not related to commit 'test: handle everything with go test'. I would create a separate commit with fixing all issues found with `go fmt` and add a job with `gofmt` to GH actions.

Reply:
separated

Reply:
Refer to https://github.com/tarantool/go-tarantool/issues/142 for a CI job